File: 800/Bear and Big Brother/babb.py
# https://codeforces.com/problemset/problem/791/A

a = int(input())    # Limak's weight
b = int(input())    # Bobs' weight
count = 1
print("The values are: ", a, "and", b)

if((a >= 1 and a <= 10) and (b >= 1 and b <= 10)):
  if a <= b:
    if a == b:
      count = 1
    else:
      while a <= b:
        a = a * 3
        b = b * 2
        if a <= b:
          count = count + 1
  print("The count is: ", count)
else:
  print("Invalid input.\n")
        

File: 800/Bear and Big Brother/babb.cpp
// https://codeforces.com/problemset/problem/791/A

#include <iostream>
using namespace std;

// Main driver
int main() {
    int a, b, count = 1;      // a is Limak's weight and b is Bob's weight
    cout << "Enter Limak's weight: ";
    cin >> a;
    cout << "Enter Bob's weight: ";
    cin >> b;
    if((a >= 1 && a <= 10) && (b >= 1 && b <= 10)) {
        if(a <= b) {
            if(a == b) {
                count = 1;
            }
            else {
                while(a <= b) {
                    a = a * 3;
                    b = b * 2;
                    if(a <= b) {
                        count++;
                    }
                }
                
            }
        }
        cout << "The count is: " << count << endl;
    }
    else {
        cout << "Invalid input.\n";
    }
    
    return 0;
}
